automatic/grafana-agent/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src="https://cdn.jsdelivr.net/gh/mkevenaar/chocolatey-packages@0462688e26372e1922bd13142072fac888eb9b3f/icons/grafana.svg" width="48" height="48"/> [grafana-agent](https://community.chocolatey.org/packages/grafana-agent)
```

Comment:
This icon needs to be updated with the package icon

Reply:
Icon is customized.